tools/topology/topology2/platform/intel/i2s-dumb-amp-echo-ref.conf

```diff
@@ -0,0 +1,73 @@
+#
+# Pcm, Dai instance & route index all start from 200 to avoid collisions
```

Comment:
Can we not use "dumb" in the name of the file please. Can you call this simply speaker-echo-ref.conf? What makes this exclusive to not-so-smart amps? When needed, the same file could be included in the case of smart amps too right?

Reply:
I agree the file name is rather dumb, for amps with IV data, we do not need the lbm_mode. Will update the file name.